--- libdom-0.0.1/Makefile~	2013-04-18 19:18:13.000000000 +0300
+++ libdom-0.0.1/Makefile	2013-04-27 17:09:02.471130386 +0300
@@ -77,6 +77,7 @@
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):$(Is)/event_listener.h
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):$(Is)/document_event.h
 
+lib := lib
 Is := include/dom/html
 I := /include/dom/html
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):$(Is)/html_document.h
@@ -97,8 +98,8 @@
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):$(Is)/html_opt_group_element.h
 INSTALL_ITEMS := $(INSTALL_ITEMS) $(I):$(Is)/html_options_collection.h
 
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib/pkgconfig:lib$(COMPONENT).pc.in
-INSTALL_ITEMS := $(INSTALL_ITEMS) /lib:$(OUTPUT)
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib)/pkgconfig:lib$(COMPONENT).pc.in
+INSTALL_ITEMS := $(INSTALL_ITEMS) /$(lib):$(OUTPUT)
 
 ifeq ($(WITH_LIBXML_BINDING),yes)
   REQUIRED_PKGS := $(REQUIRED_PKGS) libxml-2.0
